spec/system/user/user_edits_profile_spec.rb

```diff
@@ -15,12 +15,6 @@
            label: "Hate Ice Cream Flavor",
            display_area: "header")
   end
-  let!(:settings_only_profile_field) do
```

Comment:
I completely removed this because we previously had 3 distinct display areas and this spec tested 1 field in each area. Now that we only have 2 areas, 2 fields are enough.